src/utils/certificates_generator.py

```diff
@@ -21,20 +31,19 @@ def __init__(self):
         self.token = self.api_helper.auth_token
         self.header = {'Authorization': 'bln type=session,version=1,token="{}"'.format(self.token)}
 
-    @staticmethod
-    def get_worker_id_from_db():
-        db_conn = DbConnect()
-        worker_id = db_conn.fetch_all('''select "id" from "worker" where "employerId"=201 and "archived"=false;''')
-        return [x[0] for x in worker_id]
+    def get_workers_id(self):
+        if Config.env == "prod":
+            workers_list = self.api_helper.do_get_request(self.get_id_url).json()
+            return [d.get('id') for d in workers_list]
+        else:
+            db_conn = DbConnect()
+            worker_id = db_conn.fetch_all('''select "id" from "worker" where "employerId"=202 and "archived"=false;''')
```

Comment:
remove hardcoded employerID, move it to config

Reply:
fixed